src/framegrab/grabber.py

```diff
@@ -343,8 +363,10 @@ def autodiscover(
 
             # If the input type is RTSP and rtsp_discover_modes is provided, use RTSPDiscovery to find the cameras
             if input_type == InputTypes.RTSP:
-                if rtsp_discover_mode is not None:
-                    onvif_devices = RTSPDiscovery.discover_onvif_devices(auto_discover_mode=rtsp_discover_mode)
+                if rtsp_discover_mode is not AutodiscoverMode.off:
```

Comment:
the default value for rtsp_discover_mode is intending to disable autodiscover, but it was actually still happening b/c of the bad comparison to None